src/app/app.css

```diff
@@ -1,7 +1,10 @@
 :host {
-  
-  & section {
+  & {
+    min-height: 100vh;
```

Comment:
this value seems large.  when I view it, there's a giant gap in the header, not sure if that was expected?
![screen shot 2018-11-11 at 4 27 23 pm](https://user-images.githubusercontent.com/895923/48318440-2960c980-e5cf-11e8-9f9c-aa33a48d3d4b.png)

Reply:
Added a gradient as you suggested, with a css text-shadow effect, moved the header up slightly, before it was aligned in the middle of the page. 

![evergreen_style](https://user-images.githubusercontent.com/1429612/48394246-c3fcfd80-e6e0-11e8-9342-8200e6133170.png)